pkg/istio/authorizationpolicy/controller.go

```diff
@@ -55,18 +57,17 @@ func NewController(configStoreCache model.ConfigStoreCache, serviceIndexInformer
 		adIndexInformer:          adIndexInformer,
 		authzpolicyIndexInformer: authzpolicyIndexInformer,
 		queue:                    queue,
-		rbacProvider:             rbacv2.NewProvider(dryRun, enableOriginJwtSubject),
-		apResyncInterval:         apResyncInterval,
-		enableOriginJwtSubject:   enableOriginJwtSubject,
-		dryRun:                   dryRun,
+		// TODO: fix methods in rbacv2 provider funcs, find a way to check dry run flag for current ns / svc
```

Comment:
Can we remove this comment now? We do have the dry run check now which will use the corresponding event handler.